tests/test-cases/atmos-functions.yaml

```diff
@@ -18,12 +18,11 @@ tests:
       os: !not windows
     expect:
       exit_code: 0
+      # This will output in stdout because of the way our tty simulator works.
       stdout:
         - "Fetching baz output from component-3 in nonprod"
         - "Fetching foo output from component-1 in nonprod"
         - "Fetching bar output from component-2 in nonprod"
```

Comment:
Oh wait! This is wrong. 

This test should not be passing!

https://github.com/cloudposse/atmos/pull/1103/files#diff-7aee27a48716f122e74aa8a00bbd13f49e149838fdb31506e2bd48ad147d4f1eR2-R25

We should not be outputting to `stdout`

Reply:
> This will output in stdout because of the way our tty simulator works.

Nevermind. This is why it's going to `stdout`.